12LessonTwelve/homework/css/style.css

```diff
@@ -0,0 +1,257 @@
+  margin: 50px 5% 25px 5%;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+section {
+  margin: 20px 5% 20px 5%;
+  border: 1px solid lightGray;
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.topart {
+  background-image: url("http://www.thinkgeek.com/images/products/zoom/1d16_medieval_knight_hoodie.jpg");
+  background-size: 100%;
+  height: 290px;
+  background-position: center;
+  width: 100%;
+  background-repeat: no-repeat;
+  border: 1px solid lightGray;
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+
+}
+
+h2 {
+  font-size: 20px;
+  height: 30px;
+  width: 25%;
+  padding-top: 5px;
+  text-align: center;
+  background-color: lightBlue;
+  color: white;
+}
+
+h2:hover {
+  background-color: black;
+}
+
+.twoprice {
+  width: 50%;
+}
+
+.bottomart {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+h3 {
+  border: 1px solid lightGray;
+  font-size: 20px;
+  width: 100%;
+  height: 35px;
+  text-align: center;
+  padding-top: 10px;
+}
+
+button {
+  padding: 10px;
+  font-size: 20px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  color: white;
+  background-color: black;
+  border: none;
+}
+
+button:hover {
+  background-color: lightBlue;
+}
+
+h4 {
+  background-color: red;
+  color: white;
+  font-size: 20px;
+  padding: 5px;
+  display: flex;
+  margin-right: 30%;
+  height: 22px;
+  margin-bottom: 260px;
+}
+
+@media screen and (min-width: 320px) {
+  .container::before {
+    content: "320px";
+    position: absolute;
+    color: green;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+}
+
+@media screen and (min-width: 480px) {
+  .container::before {
+    content: "480px";
+    position: absolute;
+    color: green;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+
+  .topart {
+    height: 450px;
+  }
+
+  h4 {
+    margin-right: 35%;
+    margin-bottom: 420px;
+  }
+}
+
+@media screen and (min-width: 620px) {
+  .container::before {
+    content: "620px";
+    position: absolute;
+    color: green;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+
+  main {
+    display: flex;
+    flex-direction: row;
+    flex-flow: wrap;
+  }
+
+  section {
+   width: 40%;
+   display: flex;
+   margin: 2%;
+   height: 400px;
+  }
+
+  h4 {
+    margin-bottom: 217px;
+    margin-right: 23%;
+  }
+
+  h3 {
+    margin-top: -1px;
+    padding-bottom: 10px;
+  }
+
+  button {
+    height: 160px;
+  }
+}
+
+@media screen and (min-width: 810px) {
+  .container::before {
+    content: "810px";
+    position: absolute;
+    color: green;
+    top: 0;
+    left: 0;
+    font-weight: 700;
+    margin: 5px;
+    font-size: 2em;
+  }
+
+  .container {
+    max-width: 810px;
```

Comment:
I like the `max-width`; consider adding `margin: 0 auto` to keep the responsive feel though.